src/main.qml

```diff
@@ -57,6 +58,14 @@ Application {
         }
     }
 
+    function convertTempRaw(val) {
```

Comment:
I think these two functions should be reworked a little bit for clarity and maintainability, eg:

```
    function kelvinToTemperatureNumber(kelvin) {
        var celsius = (kelvin-273);
        if(!useFahrenheit.value)
            return celsius;
        else
            return Math.round(((celsius)*9/5) + 32);
    }

    function kelvinToTemperatureString(kelvin) {
        var temperatureNumber = kelvinToTemperatureNumber(kelvin);
        if(!useFahrenheit.value)
            return temperatureNumber + "°C";
        else
            return temperatureNumber + "°F";
    }
```